README.md

```diff
@@ -330,6 +330,7 @@ Not all options are turned on by default. Refer to `Tomb1Main_ConfigTool.exe` fo
 - fixed incorrect ceiling textures in Colosseum 
 - fixed inconsistencies in some enemy textures
 - fixed the animation of Lara's left arm when the shotgun is equipped
+- fixed room texture issues throughout the game, including Unfinished Business
```

Comment:
Can we list the individual fixes?

Reply:
Good point, I've updated it [here](https://github.com/LostArtefacts/Tomb1Main/pull/831/commits/d09bd005a3e0a04cbf57c17c39151d46c4c84e69).

Reply:
I mean unless there's really a lot of them, I had in mind listing every change like so:

- Lost Valley: fix black triangle in the tunnel between the lake and the caves
- Colosseum: fix Midas textures appearing at the roof

or, (this can be automated), like so:

- Lost Valley: fix missing texture in room 58
- Colosseum: fix wrong texture in rooms 12, 57, 94 and 95

Reply:
Ah, I see, sorry. I'll put a list together, there are several so I think the second approach will be best.

Reply:
A much more detailed list [here](https://github.com/LostArtefacts/Tomb1Main/pull/831/commits/29eed02c4979696821b65e6ecb7ebb2f0e13238d). I summarised it down to two types - `incorrect` to cover actual wrong textures alongside wrong rotations, and `missing` to cover actual missing faces and poorly aligned vertices that result in gaps. Then there is just the standalone Colosseum roof fix, which I've merged in manually. Let me know if this looks OK.